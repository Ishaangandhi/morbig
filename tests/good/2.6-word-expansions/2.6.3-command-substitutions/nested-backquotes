#!/bin/sh

echo `echo hello1`
echo `echo \`echo hello2\``
echo `echo \`echo \\\`echo hello3\\\`\``
