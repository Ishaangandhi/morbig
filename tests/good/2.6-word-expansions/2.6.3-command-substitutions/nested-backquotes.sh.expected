[
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName_CmdSuffix",
                [ "CmdName_Word", [ "Word", "echo" ] ],
                [ "CmdSuffix_Word", [ "Word", "`echo hello1`" ] ]
              ]
            ]
          ]
        ]
      ]
    ]
  ],
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName_CmdSuffix",
                [ "CmdName_Word", [ "Word", "echo" ] ],
                [ "CmdSuffix_Word", [ "Word", "`echo \\`echo hello2\\``" ] ]
              ]
            ]
          ]
        ]
      ]
    ]
  ],
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName_CmdSuffix",
                [ "CmdName_Word", [ "Word", "echo" ] ],
                [
                  "CmdSuffix_Word",
                  [ "Word", "`echo \\`echo \\\\\\`echo hello3\\\\\\`\\``" ]
                ]
              ]
            ]
          ]
        ]
      ]
    ]
  ],
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName_CmdSuffix",
                [ "CmdName_Word", [ "Word", "echo" ] ],
                [
                  "CmdSuffix_Word",
                  [
                    "Word",
                    "`echo \\`echo \\\\\\`echo \\\\\\\\\\\\\\`echo hello4\\\\\\\\\\\\\\`\\\\\\`\\``"
                  ]
                ]
              ]
            ]
          ]
        ]
      ]
    ]
  ]
]